plugins/plugin-maven/che-plugin-maven-server/src/main/java/org/eclipse/che/plugin/maven/server/core/MavenWorkspace.java

```diff
@@ -110,9 +110,16 @@ public void projectUpdated(Map<MavenProject, MavenProjectModifications> updated,
                 List<MavenProject> needResolve = manager.findDependentProjects(allChangedProjects);
                 needResolve.addAll(updated.keySet());
 
+                List<String> updatedPaths = updated.keySet().stream()
```

Comment:
To be compliant to streaming ideology I think it would be nice if we could have here something like this:
```
updated.keySet().stream()
                             .map(project -> project.getProject())
                             .map(project -> project.getFullPath())
                             .map(fullPath -> fullPath.toOSString())
                             .collect(Collectors.toList());
```
Don't you agree?